tests/foreman/cli/test_host.py

```diff
@@ -2207,7 +2207,7 @@ def setUpClass(cls):
             u'lifecycle-environment-id': cls.env['id'],
             u'activationkey-id': cls.activation_key['id'],
             u'subscription': cls.subscription_name,
-        })
+        }, force_use_cdn=True)
```

Comment:
we have already fixed that on 6.2 , but we missed to do it in 6.3 
https://github.com/SatelliteQE/robottelo/blob/6.2.z/tests/foreman/cli/test_host.py#L1424